README.md

```diff
@@ -139,7 +141,8 @@ the `op` CLI like so:
 
 ```sh
 cat > creds.env <<EOF
-GITHUB_WEBHOOK_SECRET = op://Bass/webhook-secret/password
+GITHUB_APP_ID = 12345
```

Comment:
I wasn't sure how to use the flags in development (do the flags still work with bud?)

So I needed to figure out what was required in `cfg`. This adds the approach env vars as an example.

Reply:
Oops, those flags are all gone now; this was written before I started using Bud. I just switched everything to env vars. I can update the rest of the README. Unless Bud lets you register flags somehow?